contribution/translation/prevalidator.php

```diff
@@ -31,18 +38,8 @@ class prevalidator
 	const REQUIRED_EMPTY = 2;
 	const REQUIRED_DEFAULT = 3;
 
-	/**
-	 * Array of files to ignore in language pack
-	 *
-	 * @var array
-	 */
-	protected $ignore_files = array(
-		'language/en/AUTHORS',
-		'language/en/README',
-		'language/en/LICENSE',
-		'language/en/CHANGELOG',
-		'language/en/VERSION',
-	);
+	const BRITISH_ENGLISH = 'british_english_3_2_5';
```

Comment:
I suppose you could have a function that uses the finder object to look inside the language_packages dir, searches for `british_english_*.zip` sorts and and chooses the latest one (or only one). So we just have to make sure the latest package is there. Then you wouldn't need this constant or need to change it every update.

Reply:
Yeah I think that's about all we can do isn't it. Added @ aad9a5c1d4d1c4573d506dfefc9c5e8f22898650